Deployment/ConnectedVehicleARMTemplate.json

```diff
@@ -256,7 +256,23 @@
                             "sku": {
                                 "name": "[parameters('StorageAccountType')]"
                             },
-                            "kind": "Storage"
+                            "kind": "StorageV2",

```

Comment:
The role assignments at the end of this file don't include an assignment for the AMI to access the storage account. It should. Don't quote me on it, but something like this should suffice:

```
        {
            "type": "Microsoft.Authorization/roleAssignments",
            "apiVersion": "2020-04-01-preview",
            "name": "[guid(concat(parameters('platformAccountName'), parameters('connectedVehicleExtensionsName'), 'contributorRole'))]",
            "scope": "[concat('Microsoft.Storage/storageAccounts/', variables('storageAccountName') )]",
            "dependsOn": [
                "[resourceId('Microsoft.Resources/deployments', 'Mcvp')]",
                "[resourceId('Microsoft.Resources/deployments', 'McvpExtensions')]"
            ],
            "properties": {
                "roleDefinitionId": "[variables('contributorRoleId')]",
                "principalId": "[reference('Mcvp').outputs.mcvpPrincipalId.value]"
            }
        }
```

Reply:
Added role assignment.